src/test/java/ru/training/at/hw2/GeneralConditions.java

```diff
@@ -0,0 +1,62 @@
+package ru.training.at.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+
+public class GeneralConditions {
+    protected WebDriver driver;
+    protected  final String homePageURL = "https://jdi-testing.github.io/jdi-light/index.html";
```

Comment:
Do you use checkStyle? There is one more space...

Reply:
And order of variables

Reply:
Yes, I use checkStyle. It didn't show any issues in this class.